src/content/docs/dropins/user-account/containers/address-form.mdx

```diff
@@ -0,0 +1,119 @@
+        ['forwardFormRef', 'HTMLInputElement', 'No', 'Provides a reference to the form\’s DOM element. Useful for custom integrations. This ref also grants access to the "handleValidationSubmit" method to validate form data.'],
+        ['addressFormId', 'string', 'No', 'Passes an address ID when editing an existing address. Used to identify the address to update in Adobe Commerce.'],
+        ['className', 'string', 'No', 'Allows custom CSS classes to be applied to the form.'],
+        ['addressesFormTitle', 'string', 'No', 'Controls visibility of the address form title.'],
+        ['inputsDefaultValueSet', 'CustomerAddressesModel', 'No', 'Allows a set of default values for form inputs.'],
+        ['shippingCheckBoxValue', 'boolean', 'No', 'Sets the initial value of the "Set as default shipping address" checkbox.'],
+        ['billingCheckBoxValue', 'boolean', 'No', 'Sets the initial value of the "Set as default billing address" checkbox.'],
+        ['showShippingCheckBox', 'boolean', 'No', 'Controls visibility of the "Set as default shipping address" checkbox. The checkbox remains in the form even if hidden, allowing shippingCheckBoxValue to set its value.'],
+        ['showBillingCheckBox', 'boolean', 'No', 'Controls visibility of the "Set as default billing address" checkbox. As with the shipping checkbox, it remains present even if hidden.'],
+        ['isOpen', 'boolean', 'No', 'Controls the form\'s visibility.'],
+        ['onSubmit', 'function', 'No', 'Allows a custom submit handler to override the default form submission behavior.'],
+        ['slots.AddressFormActions', 'AddressFormActionsContext', 'No', 'Provides an option to override the call-to-action buttons for the address form.'],
+        ['slots.AddressFormInputs', 'AddressFormInputsContext', 'No', 'Allows additional custom inputs or HTML elements at the form’s bottom.'],
+        ['slots\[`AddressFormInput_${attribute.code}`]', 'slot', 'No', 'Allows overriding input used for a particular form field. Example slot names: “AddressFormInput_firstname”, “AddressFormInput_street”.'],
+        ['onCloseBtnClick', 'function', 'No', 'Provides an option to pass a callback to close the form, useful for custom implementations where the form appears in a modal and should close upon successful submission.'],
+        ['onSuccess', 'function', 'No', 'Callback executed upon successful form submission.'],
+        ['onError', 'function', 'No', 'Callback executed if an error occurs during submission. Receives the error as a parameter.'],
+        ['onChange', 'function', 'No', 'Callback executed on each form change, receiving form data as a parameter. Useful for custom integrations.'],
+    ]}
+/>
+
+## Examples
+
+HTMLInputElement usage sample:
+
+```javascript
+const formRef = { current: null };
+
+provider.render(AddressForm, {
+  forwardFormRef: formRef,
+})(containerWrapper);
+
+const isFormValid = formRef.current.handleValidationSubmit();
+const formData = inputRef.current.formData;
```

Comment:
return {
          handleValidationSubmit,
          formData: normalizeGetAddressData(modifyFormRef, true),
          isDataValid,
        };
        
        isDataValid - A new boolean value has been added. It indicates whether the form has been validated or not without an explicit submit action.